app/controllers/idv/socure/document_capture_controller.rb

```diff
@@ -81,6 +98,32 @@ def self.step_info
           end,
         )
       end
+
+      private
+
+      def handle_stored_result
+        if stored_result&.success? && selfie_requirement_met?
+          save_proofing_components(current_user)
+          extract_pii_from_doc(current_user, store_in_session: true)
+          flash[:success] = t('doc_auth.headings.capture_complete')
+          successful_response
+        else
+          extra = { stored_result_present: stored_result.present? }
+          failure(I18n.t('doc_auth.errors.general.network_error'), extra)
+        end
+      end
```

Comment:
this function is nearly identical across all 4 document capture controllers .... seems like it should be in a concern?